kuma/install.go

```diff
@@ -49,17 +50,25 @@ func (h *handler) installKuma(del bool) (string, error) {
 }
 
 // installSampleApp installs and creates a sample bookinfo application up and running
-func (h *handler) installSampleApp() (string, error) {
+func (h *handler) installSampleApp(name string) (string, error) {
 	return "deployed", nil
```

Comment:
could such status values be centralized so that all adapters use the same terms?

Reply:
Yes @mgfeller , we need a helm chart in place for this. Inorder to phase that change we can leave it this way for the time being? Is that ok?

Reply:
Of course :-)